product_docs/docs/epas/16/upgrading/minor_update/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Minor version update with package manager"
+description: "Describes using a package manager to perfom a minor version update of EDB Postgres Advanced Server."
+navigation:
+- 05_performing_a_minor_version_update_of_an_rpm_installation
+- performing_a_minor_version_update_of_Deb_installation
+- 06_using_stackbuilder_plus_to_perform_a_minor_version_update
+---
+
+Minor version upgrades are fairly simple and only require updating existing executables and restating the database server. 
+
+Select the minor update method according to the underlying operating system of your database server:
+
+- If you installed an RPM package of EDB Postgres Advanced Server (on RHEL, AlmaLinux, Rocky Linux) with `dnf`, see [Performing a minor version update of an RPM installation](05_performing_a_minor_version_update_of_an_rpm_installation).
+
+- If you installed a Debian package of EDB Postgres Advanced Server (on Ubuntu, Debian) with `apt-get`, see [Performing a minor version update of a Debian installation](performing_a_minor_version_update_of_Deb_installation).
+
+- If you installed EDB Postgres Extended Server with the Stack Builder Plus, see [Using StackBuilder Plus to perform a minor version update](06_using_stackbuilder_plus_to_perform_a_minor_version_update).
```

Comment:
There are no EDB Postgres EXTENDED Windows installers so we can remove this ( line no 18)